tests/platform_tests/test_cpu_memory_usage.py

```diff
@@ -39,6 +39,8 @@ def setup_thresholds(duthosts, enum_rand_one_per_hwsku_hostname):
     if duthost.facts['platform'] in ('x86_64-arista_7050_qx32', 'x86_64-kvm_x86_64-r0', 'x86_64-arista_7050_qx32s',
                                      'x86_64-cel_e1031-r0', 'x86_64-arista_7800r3a_36dm2_lc') or is_asan:
         memory_threshold = 90
+    if duthost.facts['platform'] in ('x86_64-mlnx_msn4600c-r0'):
+        memory_threshold = 62
```

Comment:
From the test log, the highest memory usage is around `62.3%`. So I suggest increasing the threshold to 65%